openedx/core/djangoapps/user_api/accounts/permissions.py

```diff
@@ -23,3 +24,12 @@ class CanRetireUser(permissions.BasePermission):
     """
     def has_permission(self, request, view):
         return request.user.has_perm('accounts.can_retire_user')
+
+
+class CanReplaceUsername(permissions.BasePermission):
+    """
+    Grants access to the Username Replacement API for anyone in the group,
```

Comment:
Comment references group check but it only checks service worker user

Reply:
Fixed here and in creds, disco, and ecom.